### PR TITLE
Theme improvements

### PR DESCRIPTION
**Pull request by bdero**
- Removed unnecessary template overrides
- Removed footer links
- Removed header marketing links
- Added Sysadmin dashboard link
- Remove includes from header template for modules that don't exist in dogwood
- Header links always show when authenticated
- Ignore CSS

CC @blarghmatey @pdpinch

### CODE REVIEW

File: lms/templates/header.html [hunk 1]
```diff
@@ -6,7 +6,6 @@
 from django.core.urlresolvers import reverse
 from django.utils.translation import ugettext as _
 
-from openedx.core.djangolib.markup import HTML, Text
```
Comment (pdpinch) [inline lms/templates/header.html]: what was this?

Comment (blarghmatey) [inline lms/templates/header.html]: This theme was forked from master, but the djangolib.markup libaray isn't present in Dogwood

File: lms/templates/header.html [hunk 2]
```diff
@@ -144,12 +138,7 @@ <h2 class="course-header">
 </header>
 % if course:
 <!--[if lte IE 9]>
-<div class="ie-banner" aria-hidden="true">${Text(_('{begin_strong}Warning:{end_strong} Your browser is not fully supported. We strongly recommend using {chrome_link} or {ff_link}.')).format(
-    begin_strong=HTML('<strong>'),
-    end_strong=HTML('</strong>'),
-    chrome_link=HTML('<a href="https://www.google.com/chrome" target="_blank">Chrome</a>'),
-    ff_link=HTML('<a href="http://www.mozilla.org/firefox" target="_blank">Firefox</a>'),
-)}</div>
+<div class="ie-banner" aria-hidden="true">${_('<strong>Warning:</strong> Your browser is not fully supported. We strongly recommend using {chrome_link} or {ff_link}.').format(chrome_link='<a href="https://www.google.com/chrome" target="_blank">Chrome</a>', ff_link='<a href="http://www.mozilla.org/firefox" target="_blank">Firefox</a>')}</div>
```
Comment (pdpinch) [inline lms/templates/header.html]: Is this change part of removing djangolib.markup ?

**Comment by blarghmatey** [comment]
👍

File: lms/templates/header.html (hunk 2 above)
Comment (bdero) [inline lms/templates/header.html]: Yes

**Pull request merged**